Makefile

```diff
@@ -18,7 +18,8 @@ KAFKA_NAMESPACE ?= "kafka"
 ES_OPERATOR_NAMESPACE = openshift-logging
 ES_OPERATOR_BRANCH ?= release-4.1
 ES_OPERATOR_IMAGE ?= quay.io/openshift/origin-elasticsearch-operator:4.1
-SDK_VERSION=v0.8.1
+SDK_VERSION=v0.10.0
+GOPATH ?= "$(HOME)/go"
```

Comment:
why is this set here?

Reply:
Because of the installed tools. `go get ...` installs it under GOPATH/go/bin , and we use that later on when running golint/gosec.